test-apps/display-test-app/ios/imodeljs-test-app/imodeljs-test-app/ViewController.swift

```diff
@@ -10,103 +10,84 @@ import IModelJsNative
+            webView.topAnchor.constraint(equalTo: self.view.topAnchor),
+        ])
         self.view.setNeedsLayout()
     
-        let host = IModelJsHost.sharedInstance();
-        
-//        let frontProvider = "http://192.168.18.38:3000";
-
-        let frontProvider = "imodeljs://app";
+        let host = IModelJsHost.sharedInstance()
+        var queryParam = "#port=\(host.getPort())&platform=ios"
 
-        var queryParam = String(format: "#port=%u&platform=ios", host.getPort());
-        if (bimFile != nil) {
+        if let bimFile = bimFile {
             // Note: URL strings probably allow other characters, but we know for sure that these all work.
             // Also, we can't use `CharacterSet.alphanumerics` as a base, because that includes all Unicode
             // upper case and lower case letters, and we only want ASCII upper case and lower case letters.
             // Similarly, `CharacterSet.decimalDigits` includes the Unicode category Number, Decimal Digit,
             // which contains 660 characters.
             let allowedCharacters = CharacterSet(charactersIn: "ABCDEFGHIJKLMNOPQRSTUVWXYZabcdefghijklmnopqrstuvwxyz0123456789-_.")
-            let encodedPath = bimFile?.path.addingPercentEncoding(withAllowedCharacters: allowedCharacters)!
-            queryParam.append("&standalone=true");
-            queryParam.append("&iModelName=" + encodedPath!);
+            if let encodedPath = bimFile.path.addingPercentEncoding(withAllowedCharacters: allowedCharacters) {
+                queryParam.append("&standalone=true&iModelName=" + encodedPath)
+            }
         }
 
-        let url = URL.init(string: frontProvider + queryParam);
-        let req = URLRequest(url: url!);
-        self.webView!.load(req)
-        host.register(self.webView!);
+        webView.load(URLRequest(url: URL(string: "imodeljs://app" + queryParam)!))
+        host.register(webView)
     }
+    
     /// Show alert for webkit alert
     func webView(_ webView: WKWebView, runJavaScriptAlertPanelWithMessage message: String, initiatedByFrame frame: WKFrameInfo, completionHandler: @escaping () -> Void) {
-        
-        let alert = UIAlertController(title: message, message: nil, preferredStyle: .alert);
-        alert.addAction(UIAlertAction(title: "OK", style: .cancel,
-                                      handler: {
-                                        action in completionHandler();
-        }));
-
-        self.present(alert, animated: true);
-      }
+        let alert = UIAlertController(title: message, message: nil, preferredStyle: .alert)
+        alert.addAction(UIAlertAction(title: "OK", style: .cancel) { action in
+            completionHandler()
+        })
+        self.present(alert, animated: true)
+    }
+    
     @IBAction func onOpenSnapshotIModel(_ sender: Any) {
-        pickSnapshot();
+        pickSnapshot()
     }
+    
     func pickSnapshot(){
```

Comment:
While you're here, you should put a space before the open brace ({).

Reply:
Fixed